packages/api/schema/mutation.js

```diff
@@ -924,6 +924,19 @@ export default [
         orderPaymentId: ID!
         transactionContext: JSON
       ): String!
+
+      prepareProductMediaUpload(mediaName: String!): MediaUploadTicket!
```

Comment:
As discussed, we should simplify this, I'm still not happy with the additional API's:

- Instead of having 2 mutations per media type, we should have 1 per type and 1 that does exactly the same as the webhook

maybe something like:

```
prepareProductMediaUpload(mediaName: String!, productID: ID!): MediaUploadTicket!
prepareAssortmentMediaUpload(mediaName: String!, assortmentID: ID!): MediaUploadTicket!
prepareUserAvatarUpload(mediaName: String!, userId: ID): MediaUploadTicket!
confirmMediaUpload(mediaUploadTicketId: ID!, url: String!, size: Int!, type: String!): Media!
```

confirmMediaUpload is basically running the same code as the webhook, so if the webhook is not in action, a user can manually call it after putting but if no webhook is there, that call is not needed.